source/ways-of-working/pull-requests.html.md.erb

```diff
@@ -0,0 +1,158 @@
+---
+title: Pull requests
+last_reviewed_on: 2019-02-08
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+As a PR writer I will:
+
+- keep PRs small
+
+As a PR reviewer I will:
+
+- make an effort to review as soon as I get a second.
+- approve, as long as it’s better than before
+- favour suggestions over rejections, particularly when a multipart PR is indicated
+
+We practise code review on all changes at HMCTS to:
+
+- improve code quality
+- get team buy in on changes
+- validate change correctness
+- knowledge sharing
+
+Pull requests should not be opened when everything is finished.
+They should be opened as soon as you have something to show.
+
+Leaving it to the last minute leaves a lot of pressure to the reviewer, 
+often someone is told the code is done but they aren’t 
+told that the review hasn’t happened, pressure comes on to ship the code but there’s design problems
+ in it that need addressing, this could take hours, or days.
+
+## Size
+The smaller your pull request the easier it is to review, to test and the safer it is to send to production.
+Separating your changes out is a great skill to practice, some tips:
+
+* don’t make unrelated changes in a PR, you should still make the change if you think it improves the code
+Just take the 10 minutes to create a new branch and open a separate PR, it can be quickly merged and you can continue with your task
```

Comment:
Also, it's often important to explain if there are existing limitations of time or current way things are setup especially  if  the author had to compromise / introduce any limitations during  a change. One can always raise a ticket into backlog for fixing and mentioning in the PR helps in avoiding some obvious suggestions.

Reply:
done